packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -273,15 +277,20 @@ function _Autocomplete<ItemType>(
               </div>
             ))}
 
-          {!isLoading && items.length === 0 && (
+          {!isLoading && isShowingNoMatches && (
             <div className={cx(styles.item, styles.disabled)}>
```

Comment:
```suggestion
            <div className={styles.item}>
```
This should fix the styling on the text